Makefile

```diff
@@ -8,16 +8,10 @@ FS_MODULES=/usr/lib/freeswitch/mod
 SHELL := /bin/bash
 PROC?=$(shell uname -m)
 
-CC=gcc 
 CFLAGS=-fPIC -O3 -fomit-frame-pointer -fno-exceptions -Wall -std=c99 -pedantic
-ifeq (${PROC},x86_64)
```

Comment:
If you want to separate between 32 and 64 we could use `getconf LONG_BIT` this should return 64 or 32 based on the host OS architecture. 
But lets think about what we are doing to the CFLAGS here, basically we are only specifying options for 64bit systems that are already set if you use a 64bit compiler. For 32bit systems we force the arch to i686 but why do we do that ? Whats the reasoning behind this ?